apps/test/unit/templates/teacherDashboard/SectionActionDropdownTest.js

```diff
@@ -60,7 +61,8 @@ const DEFAULT_PROPS = {
   onEdit: () => {},
   removeSection: () => {},
   toggleSectionHidden: () => {},
-  updateRoster: () => {}
+  updateRoster: () => {},
+  setRosterProvider
```

Comment:
because `componentDidMount` now gets called by `shallow()`?

Reply:
Yes.  It's possible the safer fix here would be to disable lifecycle method calls for these tests, but this got things passing so I think it's okay.